source/application/DbgHelp/DbgHelp.h

```diff
@@ -20,8 +20,10 @@ namespace DbgHelp
     void AddProcess(HANDLE process, DWORD process_id);
     void RemoveProcess(DWORD process_id);
 
-    void LoadSymbols(DWORD process_id, HANDLE module_file, LPCWSTR module_name, DWORD64 module_base);
+    void
+    LoadSymbols(DWORD process_id, HANDLE module_file, LPCWSTR module_name, DWORD64 module_base);

```

Comment:
Priority on parameters breaking in this case if it's possible.

Reply:
Done.